### ISSUE TITLE
Enabling Application Insights for .NET Core 3.1 (LTS)

### ISSUE THREAD

**Issue by erikvanappeldoorn** [MicrosoftLearning/AZ-204-DevelopingSolutionsforMicrosoftAzure#105]
Lab 12 Exercise 1 Task 3

I am unable to enable Application Insights with the current proposed settings. In the monitoring tab an error will show up:

Application Insights is not supported for your current selection of Runtime Stack, Operating System, Publish Type, or Resource Group.

The only way I could enable Application Insights was by switching back to the .NET Core 2.1 (LTS) runtime stack.

**Comment by adnkai** [comment]
Enabling Application Insights _after_ App Service creation also worked for me.

**Comment by seesharprun** [comment]
I'm not seeing the same error. Which region are you using?

Here's screenshots of a walkthrough I did today:

1. ![image](https://user-images.githubusercontent.com/5067401/91919185-17b05080-ec93-11ea-90ee-0b31d9957b7d.png)
1. ![image](https://user-images.githubusercontent.com/5067401/91919189-1a12aa80-ec93-11ea-827e-203b289ed6bf.png)

After those steps, I was able to deploy the Web App successfully and look at the Metrics blade.
